src/lib/transform.test.js

```diff
@@ -197,6 +197,17 @@ describe('activityReportToCsvRecord', () => {
     expect(output).toMatchObject(expectedOutput);
   });
 
+  it('removes HTML tags from context', async () => {
+    const report = ActivityReport.build({
+      context: '<p data-attribute="test">test text</p>',
```

Comment:
maybe a few more test might be a good idea, order list, nested  ordered list, unordered list, and table.